src/include/utime.h

```diff
@@ -32,6 +32,10 @@
 // --------
 // utime_t
 
+//from __u64 to __u32
```

Comment:
@kungf could you drop this comment. it's unnecessary and redundant here.

Reply:
ok